Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/packit/packit
+LABEL maintainer="Ansible <info@ansible.com>"
+
+# AS packit seems to not install BuildRequires by default:
+RUN dnf install -y --setopt=install_weak_deps=False python3-wheel python3-setuptools_scm+toml python3-tox-current-env tox
```

Comment:
Why not publish an image with this pre-cached?

Reply:
As for how RPM build is supposed to work, installing the packages directly to the environment seems quite weird.  Packit offers a Copr-based service where it builds the packages for various different Fedoras/ELs. The BuildRequires are supposed to be installed for that chroot (in Copr). This seems to be a hybrid setup that tries to use packit to build the RPM package in dcoker. It might work, but it will not serve as a check whether this is packageable for Fedora, because in Fedora, we don't build packages like this.

Reply:
Oh, I missed this point. Fair enough. This container is unnecessary then and the base could just be used with a volume mount.